rest_framework/authtoken/management/commands/drf_create_token.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from rest_framework.authtoken.models import Token
+
+
+class Command(BaseCommand):
+    help = 'Create DRF Token for a given user'
+
+    def create_user_token(self, username):
+        user = User.objects.get(username=username)
+        token = Token.objects.get_or_create(user=user)
```

Comment:
I also would like to handle the situation where the user want to regenerate the Token. Should this be handled as a separate manage.py command, as an additional argument or should the command be interactive and inform the user that the Token already exist and to confirm Y/N they want to create a new Token?